src/__main__.py

```diff
@@ -34,23 +48,29 @@ def main(params):
     allClusterInstancesSheetName = "All Cluster Instances"
     allInstancesCostSheetName = "Instances Cost"
     CostSummarySheetName = "Cost Summary"
+    oldClustersSheetName = "Old Clusters"
     
     
     allInstancesSheet = GoogleSheetEditor(sheet_id, allInstancesSheetName)
     allClustersInstancesSheet = GoogleSheetEditor(sheet_id, allClusterInstancesSheetName)
     allInstancesCostSheet = GoogleSheetEditor(sheet_id, allInstancesCostSheetName)
     allClustersSheet = GoogleSheetEditor(sheet_id, allClustersSheetName)
     allSummarySheet = GoogleSheetEditor(sheet_id, CostSummarySheetName)
+    oldClustersSheet = GoogleSheetEditor(sheet_id, oldClustersSheetName)
     
+    old_clusters = get_old_clusters_data(allClustersSheet, None, tdelta=datetime.timedelta(days=0))
+
     print(allInstancesSheet.save_data_to_sheet(instances))
     print(allClustersInstancesSheet.save_data_to_sheet(cluster_instances))
     print(allInstancesCostSheet.save_data_to_sheet(cost))
     print(allSummarySheet.save_data_to_sheet(account_summary))
     print(allClustersSheet.save_data_to_sheet(clusters))
+    print(oldClustersSheet.save_data_to_sheet(old_clusters))
```

Comment:
As it currently stands, to my understanding the data in the sheet is clobbered and replaced entirely with new data. 2 problems:
- There needs to be a "Saved" column so the people can mark their clusters as saved so they don't get deleted
- When the sheet is clobbered, the data in that column will get replaced